spec/views/pages/privacy_policy.html.slim_spec.rb

```diff
@@ -7,10 +7,19 @@
     render
   end
 
-  it 'includes a link to create a new account' do
+  it 'links to the system of record notices' do
+    pending 'having the correct link'
+
+    render
+
+    doc = Nokogiri::HTML(rendered)
+    expect(doc.xpath("//a[text()='systems of records notices']").first[:href]).to_not eq('#')
+  end
+
+  it 'links to the contact form' do
     render
 
-    expect(rendered).
-      to have_link(t('links.create_account'), href: new_user_registration_path)
+    doc = Nokogiri::HTML(rendered)
+    expect(doc.css("a[href='#{contact_path}']")).to be_present
```

Comment:
because I totally forgot about `have_link`, but looking it up, it appears to be a Capybara matcher, not a Rails one or a Nokogiri one?

This particular test I actually like, I wanted to make sure there's a link to the contact page, I think it's a pretty solid test because it's using the rails path helper, so if we change the URL it'll still link to the current URL